python/doc/examples/numerical_methods/optimization/plot_optimization_pagmo.py

```diff
@@ -0,0 +1,96 @@
+"""
+Multi-objective optimization using Pagmo
+========================================
+"""
+# %%
+# In this example we are going to explore optimization using OpenTURNS' `Pagmo <https://esa.github.io/pagmo2/>`_ interface.
+
+# %%
+from __future__ import print_function
+import openturns as ot
+from openturns.viewer import View
+ot.Log.Show(ot.Log.NONE)
+
+# %%
+# List available algorithms
+for algo in ot.Pagmo.GetAlgorithmNames():
+    print(algo)
+
+# %%
+# More details on Pagmo algorithms are available `here <https://esa.github.io/pagmo2/overview.html#list-of-algorithms>`_ .
+
+# %%
+# Create the problem, from ZDT test suite
+f = ot.SymbolicFunction(['x1', 'x2'], ['x1', 'var s := x1 + x2; var g := 1 + 9 * s; g * (1.0 - sqrt(x1 / g))'])
+zdt1 = ot.OptimizationProblem(f)
+zdt1.setBounds(ot.Interval([0.0] * 2, [1.0] * 2))
+
+# %%
+# Create the first generation of points
```

Comment:
I suggest the following sentence: 

_We create a sample containing 10 points randomly created within the box bounds._

This is adapted from [pygmo](https://esa.github.io/pygmo/tutorials/basic_mo_functionalities.html)